containers/signing-service/clean-container.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+temporary_dependencies=(
+
+    gcc
+    libssl-dev
+    git
+    pkg-config
```

Comment:
We should deduplicate it. It's bad to have lists like this in two places. Maybe create two text files (one for temporary dependencies and one for permanent ones) and put dependencies in them? Or a common script sourced by both?

Reply:
Done.

Reply:
The solution with `readarray` is great.

:+1: